flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/scala/expressionDsl.scala

```diff
@@ -544,6 +544,12 @@ trait ImplicitExpressionOperations {
   def overlay(newString: Expression, starting: Expression, length: Expression) =
     Overlay(expr, newString, starting, length)
 
+  /**
+    * Returns the numeric value of the leftmost character of the string str.
```

Comment:
Which `str`? You mean the `the given expression` please see other docs for the format that we use.